From: Jiri Slaby <jslaby@suse.cz>
Date: Tue, 4 May 2021 11:17:47 +0200
Subject: drw_header: fix indentation level
Patch-mainline: Submitted pr#21
References: compiler warning fix

The compiler complains about weird indentation level in drw_header.cpp:
../src/drw_header.cpp: In member function 'void DRW_Header::write(dxfWriter*, DRW::Version)':
../src/drw_header.cpp:474:5: warning: this 'else' clause does not guard... [-Wmisleading-indentation]
  474 |     else
      |     ^~~~
../src/drw_header.cpp:476:9: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the 'else'
  476 |         writer->writeString(9, "$DIMSAH");
      |         ^~~~~~

Fix this by pushing it one level left.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 src/drw_header.cpp | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/drw_header.cpp b/src/drw_header.cpp
index 4812360c6091..ac8f65f35c0a 100644
--- a/src/drw_header.cpp
+++ b/src/drw_header.cpp
@@ -473,11 +473,11 @@ void DRW_Header::write(dxfWriter *writer, DRW::Version ver){
         writer->writeInt16(70, varInt);
     else
         writer->writeInt16(70, 0);
-        writer->writeString(9, "$DIMSAH");
-        if (getInt("$DIMSAH", &varInt))
-            writer->writeInt16(70, varInt);
-        else
-            writer->writeInt16(70, 0);
+    writer->writeString(9, "$DIMSAH");
+    if (getInt("$DIMSAH", &varInt))
+	writer->writeInt16(70, varInt);
+    else
+	writer->writeInt16(70, 0);
     writer->writeString(9, "$DIMBLK1");
     if (getStr("$DIMBLK1", &varStr))
         if (ver == DRW::AC1009)
-- 
2.31.1

